misc/toml-parser/src/main/resources/Toml.g4

```diff
@@ -0,0 +1,217 @@
+grammar Toml;
+toml : expression ( newline expression )*;
+
+ALPHA : [A-Z] | [a-z];
+alpha : ALPHA | TRUE | FALSE | NAN | INF | E | UPPERCASE_T | LOWERCASE_T | UPPERCASE_Z;
+SPACE : ' ';
+HYPHEN : '-';
+PERIOD : '.';
+QUOTATION_MARK : '"';
+UNDERSCORE : '_';
+COLON : ':';
+COMMA : ',';
+SLASH : '/';
+APOSTROPHE : '\'';
+EQUALS : '=';
+HASH : '#';
+
+expression
+           : ws
+           | ws keyval ws
+           | ws table ws;
+
+// Whitespace
+
+ws : wschar*;
+wschar :   SPACE | '\t' |'\r'| '\n' ;
+
+// Newline
+
+newline : '\n' | '\r\n';
+
+// Comment
+
+COMMENT : '#' .*? ('\n' | '\r' | '\t') -> channel(2) ;
+
+//Key-Value pairs
+
+keyval : key keyvalSep val;
```

Comment:
Shall we rename this to keyVal or keyValue. same goes with keyvalSep - keyValSep or KeyValueSep

Reply:
Fixed with 46e540d